tools/build/Makefile

```diff
@@ -78,7 +78,7 @@ _WITH_NO_SUBOBJECT_BOUNDS!= grep -c __no_subobject_bounds ${HOST_INCLUDE_ROOT}/s
 .if ${_WITH_NO_SUBOBJECT_BOUNDS} == 0
 SYSINCS+=	cdefs.h
 .endif
-# Define vaddr_t for bootstrapping
+# Define ptraddr_t for bootstrapping
```

Comment:
tools/build/stdint.h guards everything with _VADDR_T_DECLARED and only defines that... should probably tidy that up, either by separating them out or nuking vaddr_t entirely and just having the ptraddr_t bits

Reply:
My plan is to remove the vaddr_t declaration once we've got all of them out of the tree and fix the guard at that time.